tests/rt.conf

```diff
@@ -170,26 +170,33 @@ RUN     | fv3_ccpp_thompson_no_aero_debug
 
 #######################################################################################################################################################################################
-# S2S tests
+# CPLD tests
 #######################################################################################################################################################################################
 
 COMPILE | CCPP=Y SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y          | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-RUN     | cpld_fv3_ccpp_mom6_cice_cmeps_1stepcold_atm_flux                                                      | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-RUN     | cpld_fv3_ccpp_mom6_cice_cmeps_2d_atm_flux                                                             | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-RUN     | cpld_fv3_ccpp_mom6_cice_cmeps_3d_atm_flux                                                             | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-RUN     | cpld_fv3_ccpp_mom6_cice_cmeps_restart                                                                 | standard | hera.intel orion.intel wcoss_dell_p3 |     | cpld_fv3_ccpp_mom6_cice_cmeps_2d_atm_flux
-RUN     | cpld_fv3_ccpp_mom6_cice_cmeps_2d_2threads                                                             | standard | hera.intel orion.intel wcoss_dell_p3 |     |
-RUN     | cpld_fv3_ccpp_mom6_cice_cmeps_2d_decomp                                                               | standard | hera.intel orion.intel wcoss_dell_p3 |     |
-RUN     | cpld_fv3_ccpp_mom6_cice_cmeps_1d_satmedmf                                                             | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-RUN     | cpld_fv3_ccpp_384_mom6_cice_cmeps_2d_atm_flux                                                         | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-RUN     | cpld_fv3_ccpp_384_mom6_cice_cmeps_2d_atm_flux_frac                                                    | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-RUN     | cpld_fv3_ccpp_384_mom6_cice_cmeps_1d_bmark_rt                                                         | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
+RUN     | cpld_control                                                                                          | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
+#RUN     | cpld_control_12h                                                                                      | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
+#RUN     | cpld_restart                                                                                          | standard | hera.intel orion.intel wcoss_dell_p3 |     | cpld_control_12h
```

Comment:
restart test is skipped for now?

Reply:
The restart test is skipped for the default (c96 1deg ocean/ice) temporarily. It is implemented instead in the c96 1/4deg ocean and ice. That set of "mx025" tests can be dropped once we're sure that c96 1-deg can reproduce.